themes/src/main/resources/theme/base/login/frontchannel-logout.ftl

```diff
@@ -0,0 +1,32 @@
+<#import "template.ftl" as layout>
+<@layout.registrationLayout; section>
+    <#if section = "header">
+        <script>
+            document.title =  "${msg("frontchannel-logout.title")}";
+        </script>
+        ${msg("frontchannel-logout.title")}
+    <#elseif section = "form">
+        <p>${msg("frontchannel-logout.message")}</p>
+        <#if frontchannelLogout.urls?has_content>
+            <ul>
+            <#list frontchannelLogout.urls[0]?split(' ') as url>
+                <li>
+                    <iframe src="${url}" style="width:1px; height:1px;"></iframe>
```

Comment:
Looks like we are not displaying anything on this page to reference the apps we are logging out.

IMO, we could create a specific bean to build this page (like we do in some other templates) so that instead of passing the list of URLs we use the client identifiers. This bean than is responsible for returning the frontend urls and any other information about the client we want, such as the display name or client id.

Does it make sense?

Reply:
Make sense